python/ray/data/tests/test_bundle_queue.py

```diff
@@ -45,28 +45,28 @@ def test_peek():
     bundle2 = _create_bundle("test2")
     queue.add(bundle2)
 
-    peeked_bundle = queue.peek()
+    peeked_bundle = queue.peek_next()
     assert peeked_bundle is bundle1
     assert len(queue) == 2  # Length should remain unchanged
 
 
 def test_pop_empty_queue():
```

Comment:
I know it's a pain, but could you rename these tests for consistency with the updated method names? I think it'd be confusing if we have a `test_pop` but there's no `pop` method